src/world/partner/sushie.c

```diff
@@ -1,42 +1,150 @@
 #include "common.h"
+#include "../src/world/partners.h"
+#include "npc.h"
 
 extern unkPartnerStruct* D_802BFDF8_320B68;
+extern s32 D_802BFEEC;
+extern f32 D_802BFEE0;
+extern void func_802BD368_31E0D8(s32, f32, f32, f32, f32, f32);
+extern void playFX_23(s32, f32, f32, f32, s32);
```

Comment:
if you include effects.h you won't need to declare this effect func. also please remove "extern" from func decls